ga/23.0.0.12/oidcProvider/Dockerfile.java11

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM websphere-liberty:kernel-java11
+FROM websphere-liberty:23.0.0.12-kernel-java11
```

Comment:
Not entirely sure about the state of this oidcProvider. But I don't see the version included with the files for previous releases (i.e. 23.0.0.11). So I don't believe this change is needed.